python/lsst/ap/association/mapApData.py

```diff
@@ -230,16 +264,30 @@ def run(self, inputCatalog, exposure):
 
         return outputCatalog
 
+    def computeSnr(self, inputRecord, outputRecord):
```

Comment:
Since the DPDD wants this to be detection SNR specifically, using the measurements in existing catalogs this way is misleading. I'd suggest simply removing this (and related SNR pieces) for now.

Reply:
What about using PeakFlux? Would that be an acceptable middle ground while not perfectly being the `detection` snr?

Reply:
As we discussed in the `ap_pipe`, I'll remove this to avoid any confusion between the what the value here means the one requested by the Ppdb.